### PR TITLE
WinRM exec script added

### PR DESCRIPTION
**Pull request by SergeyKemaev**
Python script using WinRM added as part of https://github.com/qubell/vermilion/issues/7298 to allow to use it in `commandCall`

### CODE REVIEW

**Comment by SergeyKemaev** [comment]
@dornatsky please take a look, approve, merge if ok

**Comment by dornatsky** [comment]
@DmZ take a look

File: python-scripting/scripts/winrm_execute.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        password=password,
+        server_cert_validation='ignore')
+
+    shell_id = p.open_shell()
+    command_id = p.run_command(shell_id, powershell_command)
+    std_out, std_err, status_code = p.get_command_output(shell_id, command_id)
+
+    p.cleanup_command(shell_id, command_id)
+    p.close_shell(shell_id)
+    results[ip] = {"results": std_out.decode('ascii'), "errors": std_err.decode('ascii')}
```
Comment (dornatsky) [inline python-scripting/scripts/winrm_execute.py:37]: how does one validate if the command was a success without analyzing the map?

File: python-scripting/scripts/winrm_execute.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+encoded_ps = b64encode(command.encode('utf_16_le')).decode('ascii')
+
+powershell_command = 'powershell {0} -encodedcommand {1}'.format(arguments, encoded_ps)
+
+results = {}
+
+for ip in ips:
+    p = Protocol(
+        endpoint='https://{0}:5986/wsman'.format(ip),
+        transport='ntlm',
```
Comment (dornatsky) [inline python-scripting/scripts/winrm_execute.py]: will this be different for kerberos and should we make this a parameter?

File: python-scripting/scripts/winrm_execute.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        username=r'{0}\{1}'.format(ips, username),
+        password=password,
+        server_cert_validation='ignore')
```
Comment (dornatsky) [inline python-scripting/scripts/winrm_execute.py]: is this how we want to run it?

File: python-scripting/scripts/winrm_execute.py (hunk 2 above)
Comment (SergeyKemaev) [inline python-scripting/scripts/winrm_execute.py]: Yes, but we do not want to support other authentication methods at the moment.

File: python-scripting/scripts/winrm_execute.py (hunk 3 above)
Comment (SergeyKemaev) [inline python-scripting/scripts/winrm_execute.py]: It satisfies our expectations if I understand correctly.

**Pull request merged**